### DISCUSSION TITLE
Find a cylinder Unfold diagram code

### DISCUSSION THREAD

**Discussion by xiaoxuz** [ManimCommunity/manim#3590] [Q&A]
![URL_08791b5a79774470020e674e20a0d902](https://github.com/ManimCommunity/manim/assets/11401345/0c834cfe-7b47-485e-9a2e-9c54ba95c795)

Something like this effect

**Comment by uwezi** [comment]
is this a question? a call for help?
When I saw this on reddit I thought it was the presentation of a nice animation....

**Reply by xiaoxuz** [reply]
This animation was manually created by me using another software, and now I want to implement it using Manim, so I need a piece of code.

**Comment by xiaoxuz** [comment]
This animation was manually created by me using another software, and now I want to implement it using Manim, so I need a piece of code.

**Comment by uwezi** [comment]
Use a value tracker and parametrized shapes for the cylinder wrapping and the lids. Don't expect anyone to do the full job for you.
